### PR TITLE
Porting changes to fix seek after end into stable

### PR DESCRIPTION
**Pull request by bc-bbay**

### CODE REVIEW

**Comment by bclwhitaker** [comment]
LGTM

**Comment by bclwhitaker** [comment]
@imbcmdth ?

File: src/com/videojs/providers/HTTPVideoProvider.as [hunk 1]
```diff
@@ -574,8 +573,6 @@ package com.videojs.providers{
                         _model.broadcastEvent(new VideoPlaybackEvent(VideoPlaybackEvent.ON_STREAM_CLOSE, {info:e.info}));
                         _model.broadcastEventExternally(ExternalEventName.ON_PAUSE);
                         _model.broadcastEventExternally(ExternalEventName.ON_PLAYBACK_COMPLETE);
-                        _startOffset = 0;
-                        _pausedSeekValue = 0;
```
Comment (imbcmdth) [inline src/com/videojs/providers/HTTPVideoProvider.as]: These two changes were not in the other PR.

Comment (bc-bbay) [inline src/com/videojs/providers/HTTPVideoProvider.as]: https://github.com/videojs/video-js-swf/pull/170/files

They came from this one, which has already been accepted into master. I was just consolidating the two PRs into one for the port to stable.

Comment (imbcmdth) [inline src/com/videojs/providers/HTTPVideoProvider.as]: OK. @bclwhitaker LGTM :ice_cream:

**Pull request merged**